kabob-build/src/main/resources/edu/ucdenver/ccp/kabob/build/rules/bio_labels/proteins.clj

```diff
@@ -19,21 +19,58 @@
    (_/record obo/BFO_0000051 _/typeRecordField)
    (_/typeRecordField kiao/hasTemplate iaouniprot/ProteinSchema1)
    (_/typeRecordField rdf/type iaouniprot/Protein)
-   ;; This record will also contain the recommended name of the protein.
-   (_/typeRecordField obo/BFO_0000051 _/recommendedNameRecord)
-   (_/recommendedNameRecord kiao/hasTemplate iaouniprot/RecommendedNameSchema1)
-   (_/recommendedNameRecord rdf/type iaouniprot/RecommendedName)
-   ;; The recommended name field is an evidenced string.
-   (_/recommendedNameRecord obo/BFO_0000051 _/evidencedStringRecord)
-   (_/evidencedStringRecord kiao/hasTemplate iaouniprot/EvidencedStringSchema1)
-   (_/evidencedStringRecord rdf/type iaouniprot/EvidencedString)
-   ;; The value of the evidenced string is what we want as the protein name.
-   (_/evidencedStringRecord obo/BFO_0000051 _/evidencedStringValue)
-   (_/evidencedStringValue obo/IAO_0000219 ?/proteinName))
+   ;; This record will also contain the unique name of the protein.
+   (_/record obo/BFO_0000051 _/nameRecord)
+   (_/nameRecord kiao/hasTemplate iaouniprot/UniProtFileRecord_nameDataField1)
+   (_/nameRecord obo/IAO_0000219 ?/proteinName))
 
  ;; The form of the new triples that will be constructed.
  :head
- '((?/bio rdf/label ?/proteinName))
+ '((?/bio rdfs/label ?/proteinName))
+
+ ;; In this case no `:reify` entry is required, since all of the data required
+ ;; to form the new triples already exist.
+
+ :options
+ {:magic-prefixes
+  [;; AllegroGraph processing directives; http://franz.com/agraph/support/documentation/current/sparql-reference.html#header2-138
+
+   ;; Note: DO NOT set the clause reorderer to `identity`.  The default
```

Comment:
If I had to guess, I would say that swapping lines 16 and 17 in the swiss-prot rule and 64 and 65 in the trembl rule would remedy the slow query times you are observing when using the `identity` clause reorderer. As it stands, lines 16 and 64 are completely disconnected from the beginning parts of the queries (i.e. the line contains no bound variables) and probably result in the collection of all possible records, which are then filtered in the next line based on the bound _/accessionRecordField.